giza/giza/content/options/views.py

```diff
@@ -25,9 +25,14 @@ def render_options(option, conf):
     if 'program' not in option.replacement:
         option.replacement['program'] = RstCloth.role("program", option.program)
 
-    if option.directive == 'option':
+    if option.has_field('command'):
+        option.replacement['command'] = RstCloth.role("toolcommand", option.command)
+
+    if option.directive in ['option']:
         if option.name.startswith('<'):
             prefix = ''
+        elif option.name.startswith('-'):
```

Comment:
I'd be inclined to roll this into the previous branch with an or, if it doesn't make the line too long. This implies at first glance that something different is being done.